deeprank_gnn/NeuralNet.py

```diff
@@ -19,15 +20,29 @@
 _log = logging.getLogger(__name__)
 
 
-class NeuralNet(object):
-
-    def __init__(self, database, Net,
-                 node_feature=['type', 'polarity', 'bsa'],
-                 edge_feature=['dist'], target='irmsd', lr=0.01,
-                 batch_size=32, percent=[1.0, 0.0],
-                 database_eval=None, index=None, class_weights=None, task=None,
-                 classes=[0, 1], threshold=None,
-                 pretrained_model=None, shuffle=True, outdir='./', cluster_nodes='mcl', transform_sigmoid=False):
+class NeuralNet():
+    def __init__( # noqa
+        self,
+        database,
+        Net,
+        node_feature=None,
+        edge_feature=None,
+        target="irmsd", # noqa
```

Comment:
As above, several arguments that are in the init are then now used. Should we remove them?

Reply:
What arguments do you mean?

target is used here:  https://github.com/DeepRank/deeprank-gnn-2/blob/82a2656ad55a58c32c95374b99dbfefed83daa01/deeprank_gnn/NeuralNet.py#L88

Reply:
I just merged with main, and this file now is different. I am solving conflicts now